### PR TITLE
Revamp pricing page

### PR DESCRIPTION
**Pull request by zchase**
Fixes: https://github.com/pulumi/docs/issues/4547

Further details are in the issue but this PR revamps the pricing as per the agreed-upon design. The design can be viewed [here](https://www.figma.com/file/j1Fq8UsKL7HE56jz0VoLbT/pricing-page-nov-2020-updates?node-id=0%3A1). I opted to keep the form at the bottom of the page so the link for "Schedule a Demo" stays on the page.

### CODE REVIEW

**Review by susanev — APPROVED** [review]
thanks so much for doing this work so quickly!! 🎉 
~noticed a few small things~

~- [ ] this text is slightly different on the desktop page, is that possible to do? `The Pulumi Service helps teams adopt the Open Source SDK, with easy, robust, and secure state management and workflows.` also would love if we can keep Open Source SDK all on the same line if possible~
~- [ ] change Pulumi Service for Teams to `Pulumi Service`~
~- [ ] below the service heading add `Prefer to use the service as an individual for free? Get started`~
~- [ ] change `Contact Us` back to `Custom`~
~- [ ] swap the order back to lowest cost first~

love the accordions and the contact form (thx for adding that in!)

**Comment by joeduffy** [comment]
@susanev I got a bit more feedback from a couple of folks, and two bits seem to be recurring and similar to what @cnunciato pointed out. It's making me ponder. Also curious if @jaxxstorm has feedbak.

The two points of feedback were

* Due to the font size, and ordering from enterprise-to-starter left-to-right, "Contact Us" is the first thing the eye is drawn to. This can emphasize the commercial/enterprise feel more than maybe we want to. (It used to be "Custom" which, in addition to being smaller, might land slightly better since our target audience hates talking to salespeople.)

* The lack of the individual tier on the left hand side where your eye was naturally drawn to definitely makes it feel like we don't have that free option for individuals. I explained the new phrasing and consensus was that it might not be enough to offset the initial reaction of the enterprise-forward presentation.

**Comment by susanev** [comment]
@joeduffy was hoping to a/b test the ordering of the team plans after we got this out, but we can def start with the team starter first, we can def switch back to custom too, that was nit from @zchase i believe but nothing we need to change now. ill add that to my list of feedback for @zchase 

and with regards to the individual plan, we could take a step backward and go back to an earlier iteration that didn't combine all into one action if we would like to that may be a bit less drastic and folks might be more comfortable with that step, let me know
[figma earlier iteration](https://www.figma.com/file/LDzlYOa7fFDn4AKK1kNNIY/pricing-page-iterations?node-id=17%3A0)

**Comment by susanev** [comment]
spoke with joe, we are gonna go back towards an earlier iteration. i will update the designs later today.

**Comment by susanev** [comment]
alright designs are updated
[figma designs](https://www.figma.com/file/j1Fq8UsKL7HE56jz0VoLbT/pricing-page-nov-2020-updates?node-id=0%3A1)

i think that requires these changes

- [ ] add a community card to the top
- [ ] community card button should become primary on mobile since its at the top of the page
- [ ] adjust the heading on the team starter plan, `Everything in Community...`
- [ ] remove pulumi service heading

**Comment by susanev** [comment]
@zchase everything looks good, one more small thing I noticed...
- [ ] get started button on community card on desktop view should be secondary like the one on open sdk (on mobile it becomes primary cause its at the top of the page)

**Comment by joeduffy** [comment]
One small thing that just struck me, @susanev what do you think of this? I added an "Open source for teams" that links to the section below, in case someone has the common reaction we know they have (only free options for individuals):

image

Maybe some wordsmithing to do here -- e.g., "Open source options for teams" -- but that's the general idea.

**Comment by susanev** [comment]
love it!

File: assets/sass/_pricing.scss [hunk 1]
```diff
@@ -1,31 +1,137 @@
...
 
-.pricing-item {
-    @apply flex items-center border-b border-gray-100 text-xs text-center;
+.pricing-compare-plans-table {
+    @apply mt-8;
+
+    .compare-plans-table-header {
+        &-container {
+            @apply flex px-8 py-5;
+            background-color: rgba(42, 19, 55, 0.05);
```
Comment (cnunciato) [inline assets/sass/_pricing.scss:33]: Let's make this either `bg-gray-100` or `-200`, to keep it within the palette. (Alternatively, we could extend that palette to include another hue, but I'd rather not do one-off styles like this unless they're not really representable with Tailwind.)

File: assets/sass/_pricing.scss [hunk 2]
```diff
@@ -1,31 +1,137 @@
...
 
-        @screen md {
-            @apply font-bold py-4 px-8;
+                    a {
+                        @apply text-black;
+                    }
+                }
+            }
+
+
```
Comment (cnunciato) [inline assets/sass/_pricing.scss:97]:
```suggestion
```

**Review by cnunciato — COMMENTED** [review]
Few little things we could probably tidy up, mainly on mobile, but otherwise looking great.

On desktop, the upper and lower borders don't quite line up:

image


Text overlaps in several sections on mobile:

image

The design makes fuller use of horizontal space here:

image

The design doesn't give explicit guidance for how these tabs should look on mobile, we could probably make this look a bit better with some center alignment and/or margin/padding reduction:

image

**Comment by susanev** [comment]
@zchase on mobile lets drop the tabs for the faqs and add headings instead above each group

**Comment by cnunciato** [comment]
> we should at some point get a responsive table for our website, but I think we don't have to do that in this pr

💯 🌟

**Comment by joeduffy** [comment]
@zchase Is tomorrow the day? 🤩

**Comment by zchase** [comment]
@joeduffy Yup I'm addressing the remaining feedback this morning!

**Comment by susanev** [comment]
@zchase i saw one more small thing, but also good with doing it in another pr and merging this, whatever you think is best

- [ ] open source link in community box should have the same styles as the schedule a demo one

**Review by cnunciato — APPROVED** [review]
Save for @susanev's comments, merge when ready. Looks great. 🚀 🚀 🚀

**Pull request merged**